src/_tutorials/dart-vm/httpserver.md

```diff
@@ -271,38 +270,32 @@ The form also specifies the URL, which includes the port number,
 and the kind of request (the _request method_).
 It might also include elements that build a query string.
 
-Here's the HTML code for the form in make_a_guess.html:
+Here's the form HTML from `make_a_guess.html`:
 
+<?code-excerpt "httpserver/web/make_a_guess.html (form)" replace="/(action|method|name|type)=(.).*?\2/[!$&!]/g"?>
 {% prettify html %}
-[[note]]1[[/note]] <form [[highlight]]action="http://localhost:4041"[[/highlight]]
-[[note]]2[[/note]]       [[highlight]]method="GET"[[/highlight]]>
-
-[[note]]3[[/note]]   <select [[highlight]]name="q"[[/highlight]]>
-        <option value="0">0</option>
-        <option value="1">1</option>
-        <option value="2">2</option>
-        ...
-        <option value="9">9</option>
-      </select>
-[[note]]4[[/note]]   <input [[highlight]]type="submit"[[/highlight]] value="Guess">
-    </form>
+<form [!action="http://localhost:4041"!] [!method="GET"!]>
```

Comment:
No, it is because we can't put HTML comments inside tags (which is effectively what we would need to do here). 

An alternative could be to have two markers on a single line like this:
```html
<form action="http://localhost:4041" method="GET"> <!--1--> <!--2-->
  <select name="q"> <!--3-->
    <option value="0">0</option>
    <option value="1">1</option>
    <option value="2">2</option>
    <!-- ··· -->
    <option value="9">9</option>
  </select>
  <input type="submit" value="Guess"> <!--4-->
</form>
```

That doesn't seem very readable.

Other, more elaborate solutions (e.g., involving tooltip popups) might be possible, but I didn't think that it is worth it for just a single example. WDYT?